### PR TITLE
Add Kerbal Space Program cheatsheet

### PR DESCRIPTION
**Pull request by JamyGolden**
**What does your Instant Answer do?**
Cheat sheet added.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://wiki.kerbalspaceprogram.com/wiki/Key_bindings

**Why did you choose this data source?**
It comes directly from the company that makes the game.

**Are there any other alternative (better) data sources?**
There are not.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Are you having any problems? Do you need our help with anything?**
No thanks.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
I don't think this applies to cheatsheet additions but I'll add anyway. Screenshot

-----
IA Page: https://duck.co/ia/view/kerbal_space_program_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+        "Docking mode",
+        "EVA Commands",
+        "VAB/SPH Commands"
+    ],
+    "sections": {
+        "System/UI Commands": [{
+            "key":"[←] / [→] / [↑] / [↓]",
+            "val":"Turn view L/R/U/D"
+        }, {
+            "key":"[+] (Numpad)",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: I'm not sure `(numpad)` is really necessary? I imagine users w/o a numpad can do `[Shift] [=]`?

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+    "section_order": [
+        "System/UI Commands",
+        "Flight Controls",
+        "Docking mode",
+        "EVA Commands",
+        "VAB/SPH Commands"
+    ],
+    "sections": {
+        "System/UI Commands": [{
+            "key":"[←] / [→] / [↑] / [↓]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: Let's break this up into 4 different key:value pairs

Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: Actually, maybe not. This is pretty big already...

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+            "key":"[Mod] + [F12]",
+            "val":"Open debug console"
+        }, {
+            "key":"M",
+            "val":"Orbital map toggle"
+        }, {
+            "key":"Tab",
+            "val":"Cycle focus in orbital map"
+        }, {
+            "key":"[Left-Shift] + [Tab]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: the `+` isn't necessary and should be removed. Combined keys should just have a space between them, successive key presses should be separated with a comma (`,`):

E.g. `[Left-Shift] [Tab]`

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+            "key":"X",
+            "val":"Cut throttle"
+        }, {
+            "key":"Z",
+            "val":"Full throttle"
+        }, {
+            "key":"G",
+            "val":"Toggle landing gear"
+        }, {
+            "key":"[Mod] + [W]/[S]/[A]/[D]/[Q]/[E]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: Please wrap these sub-alternatives in `()`: `"[Mod] + ([W]/[S]/[A]/[D]/[Q]/[E])"`

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+            "key":"U",
+            "val":"Toggle vehicle lights"
+        }, {
+            "key":"[1-9], [0]",
+            "val":"Activate custom action group 1-10"
+        }, {
+            "key":"Backspace",
+            "val":"Activate abort action group"
+        }, {
+            "key":"[Right-click]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: I don’t think this needs the `[]`?

File: share/goodie/cheat_sheets/kerbal-space-program.json [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+            "key":"D",
+            "val":"Walk/jet right"
+        }, {
+            "key":"Q",
+            "val":"Jet rotate left"
+        }, {
+            "key":"E",
+            "val":"Jet rotate right"
+        }, {
+            "key":"[Left-click] and drag left",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/kerbal-space-program.json]: Wondering if the `and` should be a `,`? `[Left-click], drag left`.

**Comment by moollaza** [comment]
@JamyGolden thanks a lot for contributing, we really appreciate it! I've left a few comments, overall you've done an awesome job. Thanks :smile:

**Comment by JamyGolden** [comment]
@moollaza Thanks for the comments. I've updated the things you've mentioned.

**Comment by zekiel** [comment]
https://duck.co/ia/view/kerbal_space_program_cheat_sheet

**Comment by moollaza** [comment]
@JamyGolden so sorry for the delay here! this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@JamyGolden sorry for the delay, congrats! It's live: https://duckduckgo.com/?q=Kerbal+Space+Program+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**